src/services/helpers.ts

```diff
@@ -206,16 +204,3 @@ export const getSortedAndFilteredVocabularyItems = (
 
 export const willNextExerciseUnlock = (previousScore: number | undefined, score: number): boolean =>
   score > SCORE_THRESHOLD_UNLOCK && (previousScore ?? 0) <= SCORE_THRESHOLD_UNLOCK
-
```

Comment:
i'm not sure why we reading here a text file with the reference to the cached image (which may be deleted meanwhile)

Reply:
This function is needed, as we need to load the images, for e.g. showing the VocabularyListItem  or any other component correctly. Own vocabulary should be possible to be passed and handled like lunes vocab or anything else.
This causes an error, e.g. if you open the list of own vocabulary:  `ERROR  TypeError: undefined is not an object (evaluating 'images.length')`

Reply:
can not reproduce this error. tested on real devices

Reply:
Didn't see it on android as well

Reply:
My bad, needed to clear async storage.